permissions/api_permissions/graphene_permissions.py

```diff
@@ -353,6 +354,18 @@ def has_mutation_permission(cls, root: Any, info: ResolveInfo, input: dict) -> b
         return False
 
 
+class ReservationCommentPermission(BasePermission):
+    @classmethod
+    def has_mutation_permission(cls, root: Any, info: ResolveInfo, input: dict) -> bool:
+        pk = input.get("pk")
+        if pk:
+            reservation = get_object_or_404(Reservation, pk=pk)
+            return can_handle_reservation(
```

Comment:
Should we add the can_handle helper func in the can_comment helper? Wouldn't need to combine here but would be just a straight helper function what to use here.

Reply:
Good idea! I added the helper function and simplified this ✅